### ISSUE TITLE
Issue about training a LogisticRegression model instead of SGC model.

### ISSUE THREAD

**Issue by JialeDeng** [Cyrus9721/Characterizing_graph_influence#1]
Hello! The paper was quite well-written and this work inspired me a lot, but I have some concerns about the code.
In particular, when computing the node influence, a `SimplifiedGraphNeuralNetwork` (which is on earth a LogisticRegression model) is trained instead of SGC model mentioned in the paper. The code is in https://github.com/Cyrus9721/Characterizing_graph_influence/blob/c0c87b6570b83f2d8a5ab4ed548d5cf9184cf9a1/calculate_node_influence.py#L108
It is quite confusing since the LogisticRegression model leverages no graph structure and computing influence score on them seems not an approximation to the SGC model.
I would be very grateful if you could answer the questions.
Thanks again for providing such inspiring work, and looking forward to your reply!

**Comment by Cyrus9721** [comment]
Hi Jiale,

Thank you so much for your interest in this work. Here is the explanation:

In the original SGC paper (https://arxiv.org/abs/1902.07153), section 2.2,
equation (8), the SGC can be reformulated as Y_{SGC} = softmax S^{K}XΘ. In
this case, S^{K}X can be precomputed.
Also, you can take a look at DGL's documentation at (
https://docs.dgl.ai/en/0.8.x/_modules/dgl/nn/pytorch/conv/sgconv.html#SGConv),
the code chunk for '# compute (D^-1 A^k D)^k X'.


Best,
Zizhang

**Comment by JialeDeng** [comment]
Thanks for your answer, I understand the code now. :-)
